### PR TITLE
chore: Stop being an esm module

### PR DESCRIPTION
**Pull request by chee**
Instead we'll use .mjs files. Storybook didn't like it when
its main file was in a type: module repo

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,11 +3,9 @@
 	"name": "origami",
 	"version": "1.0.0",
 	"description": "the reference implementation of the FT's design system",
-	"type": "module",
 	"scripts": {
-		"regenerate": "node ./scripts/regenerate-ci-config-files.js",
-		"test": "node ./scripts/test.js",
-		"install": "cp hooks/* .git/hooks && cd tools/origami-build-tools && npm ci"
+		"regenerate": "node ./scripts/regenerate-ci-config-files.mjs",
+    "test": "node ./scripts/test.mjs",
```
Comment (chee) [inline package.json:8]:
```suggestion
	"install": "cp hooks/* .git/hooks && cd tools/origami-build-tools && npm ci",
	"test": "node ./scripts/test.mjs"
```

File: scripts/check-next-version.mjs [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import core from "@actions/core"
+import {
+	getChangedPackages,
+	filterUserFacing,
+} from "./lib/getChangedPackages.mjs"
```
Comment (chee) [inline scripts/check-next-version.mjs:5]:
```suggestion
} from "./lib/get-changed-packages.mjs"
```

File: .github/workflows/sync-repo-labels.yml [hunk 3]
```diff
@@ -1,12 +0,0 @@
-name: Sync Origami labels
-on:
-  - issues
-  - pull_request_target
-jobs:
-  sync-labels:
-    runs-on: ubuntu-latest
-    name: Sync repository labels
-    steps:
-      - uses: Financial-Times/origami-labels@v1
```
Comment (notlee) [inline .github/workflows/sync-repo-labels.yml:10]: Looks good except was deleting this intentional?

**Review by notlee — CHANGES_REQUESTED** [review]

File: .github/workflows/sync-repo-labels.yml (hunk 3 above)
Comment (chee) [reply .github/workflows/sync-repo-labels.yml:10]: yes, our labeling strategy for this repo doesn't include `origami-labels`

**Comment by chee** [comment]
another strategy would be to move the storybook stuff into a subdirectory with its own package json

**Pull request closed without merging**